docs/units/format.rst

```diff
@@ -1,27 +1,33 @@
 .. _astropy-units-format:
 
-String representations of units
+String Representations of Units
 *******************************
 
-.. |quantity| replace:: :class:`~astropy.units.Quantity`
+.. |Quantity| replace:: :class:`~astropy.units.Quantity`
 
-.. |unit| replace:: :class:`~astropy.units.UnitBase`
+.. |Unit| replace:: :class:`~astropy.units.UnitBase`
 
-Converting units to string representations
+Converting Units to String Representations
 ==========================================
 
-You can control the way that |quantity| and |unit| objects are rendered as
+You can control the way that |Quantity| and |Unit| objects are rendered as
 strings using the new `Format String Syntax
```

Comment:
New and old here is now quite ancient. How about changing it to "`Python Format String Syntax...`" and then deleting the whole sentence after it. After all, these days one would use f-strings...